src/main/java/racingcar/Application.java

```diff
@@ -2,6 +2,7 @@
 
 public class Application {
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+        RacingGameController gameController = new RacingGameController();
+        gameController.start();
```

Comment:
```suggestion
        gameController.start();
        Console.close();
```
저도 이번에 알게 되었는데 `Console.close()`가 있더라고요! 메모리 누수를 방지하고자 하면 좋을 것 같습니다!

Reply:
오,, 그럼 저경우는 `console`로 입력받으면서 생겼던 메모리가 정리되는걸 의미하는걸까요??
근데 프로그램의 출력이 다 끝난 후에면 `console`은 종료되니까 메모리가 없어지지않나욤..??
(사실 처음봐서 궁금했습니다👀😮)